api/models/templates/app.tmpl

```diff
@@ -633,15 +634,18 @@
     },
     "Volumes": [
       {{ range .MountableVolumes }}
-        "{{ . }}",
+        {{ if eq .Host "/var/run/docker.sock" }}
```

Comment:
is this really the only non /volumes/ mount we expect going forward? I can't think of anything else we explicitly support, just asking out loud. And points to some documentation improvements obviously. Can't find anything about volumes right now.